jmx/src/main/java/org/jboss/as/jmx/JMXSubsystemAdd.java

```diff
@@ -72,6 +76,18 @@ static void launchServices(OperationContext context, ModelNode model, ManagedAud
         } else {
             isMasterHc = false;
         }
+        if (context.getProcessType() == ProcessType.HOST_CONTROLLER) {
```

Comment:
I'm going to change this some. We need to handle ProcessType.DOMAIN_SERVER; this block looks about right for that case. Handling HOST_CONTROLLER is good if this is a host-controller subsystem. But there, server-group will always be null (as an HC is not in a server group).